doc/automatic-tls.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Experimental: Automatic TLS"
+docpage = true
+[menu.docs]
+    parent = "automatic-tls"
++++
+
+As of [Conduit v0.5.0][conduit-v0.5.0], Conduit can be configured to automatically
+instrument applications to communicate with Transport Layer Security (TLS).
+
+When TLS is enabled, Conduit automatically establishes and authenticates
+secure, private connections between Conduit proxies. This is done without
+breaking unencrypted communication with endpoints that are not configured
+with TLS-enabled Conduit proxies.
+
+This feature is currently **experimental** and is designed to _fail open_ so
+that it cannot easily break existing applications. As the feature matures,
+this policy will change in favor of stronger security guarantees.
+
+### Getting started with TLS
+
+The conduit control plane must be installed with the `--tls=optional` flag:
```

Comment:
I think it's worth a sentence or paragraph somewhere that explains what `optional` signifies (possibly by contrasting against other, future modes of operation). If I came into this blind, I would expect it to be called "enabled" or "on".

Reply:
"optional" is used to indicate the "fail-open" behavior.